### PR TITLE
zeebe docs review

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Related to https://github.com/camunda/developer-experience/issues/63.

Pretty minor adjustments here, but looping in @jwulf in the case he has additional suggestions.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/zeebe/technical-concepts/internal-processing.md [hunk 1]
```diff
@@ -64,6 +64,8 @@ When the broker receives more requests than it can process with an acceptable la
 
 Backpressure is indicated to the client by throwing a **resource exhausted** exception. If a client sees this exception, it can retry the requests with an appropriate retry strategy. If the rejection rate is high, it indicates the broker is constantly under high load and you need to reduce the rate of requests. Alternatively, you can also increase broker resources to adjust to your needs. In high-load scenarios, it is recommended to [benchmark](https://camunda.com/blog/2022/05/how-to-benchmark-your-camunda-platform-8-cluster/) your Zeebe broker up front to size it correctly.
 
+<!--- Would it be worth adding documentation for benchmarking? --->
```
Comment (akeller) [inline docs/components/zeebe/technical-concepts/internal-processing.md:67]: This is the question of the moment. On our [Contact page](https://docs.camunda.io/contact/) we ask that people reach out about performance and benchmarking. I _think_ this is so we can hand hold folks through the process and ensure they get the correct setup and outcome. 

My guess is this is a PM or even consulting question. You could also follow up with support if you want to get more data on whether customers are asking for documentation on benchmarking or just asking for assistance in benchmarking.

File: docs/components/zeebe/technical-concepts/clustering.md [hunk 2]
```diff
@@ -42,6 +42,8 @@ This may lead to one node becoming the leader for all partitions. This is not a
 To reach a well-distributed leadership again, the [Rebalancing API](../../../self-managed/zeebe-deployment/operations/rebalancing.md) can be used. Be aware that this is on a best-effort basis.
 :::
 
+<!--- Is this Rebalancing API available for Camunda 8 SaaS environments, or only Self-Managed? --->
+
```
Comment (deepthidevaki) [inline docs/components/zeebe/technical-concepts/clustering.md:46]: It is not available for SaaS users. But it is available internally, and we use it sometimes in support/incidents.

File: docs/components/zeebe/technical-concepts/partitions.md [hunk 3]
```diff
@@ -26,13 +26,13 @@ When you start an instance of a process, the client library then routes the requ
...
 - When a client publishes a message to trigger a **message start event**, the message is forwarded to a partition based on the correlation key of the message. The process instance is created on the same partition where the message is published.
 - Process instances created by **timer start events** are always created on partition 1.
 
 ## Scalability
 
-Use partitions to scale your process processing. Partitions are dynamically distributed in a Zeebe cluster and for each partition there is one leading broker at a time. This **leader** accepts requests and performs event processing for the partition. Let's assume you want to distribute process processing load over five machines. You can achieve that by bootstraping five partitions.
+Use partitions to scale your process processing. Partitions are dynamically distributed in a Zeebe cluster and for each partition there is one leading broker at a time. This **leader** accepts requests and performs event processing for the partition. Let's assume you want to distribute process processing load over five machines; you can achieve that by bootstraping five partitions.
```
Comment (deepthidevaki) [inline docs/components/zeebe/technical-concepts/partitions.md:35]: Suggestion: five or more partitions. 

:thinking: We can add some content about dynamic scaling here. Probably after 8.4 release.

**Review by deepthidevaki — APPROVED** [review]
:rocket: Thanks for improving the docs.

**Review by oleschoenburg — APPROVED** [review]

File: docs/components/zeebe/technical-concepts/internal-processing.md (hunk 1 above)
Comment (christinaausley) [reply docs/components/zeebe/technical-concepts/internal-processing.md]: Inquired on Slack about this 👍

**Review by deepthidevaki — APPROVED** [review]

**Pull request merged**